src/script_interface/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ include_directories(${CMAKE_SOURCE_DIR}/src)
 include_directories(${CMAKE_SOURCE_DIR}/src/core)
 include_directories(${CMAKE_BINARY_DIR}/src/core)
 if (HDF5_FOUND)
-    include_directories(${CMAKE_SOURCE_DIR}/libs/h5xx)
+    include_directories(${CMAKE_SOURCE_DIR}/libs/h5xx SYSTEM)
```

Comment:
This will be changed back once https://github.com/h5md/h5xx/pull/1 and https://github.com/h5md/h5xx/pull/2 are merged.